src/common-middleware.js

```diff
@@ -0,0 +1,40 @@
+// Setup sharify
+// TODO: Export a function instead of loading on import.
+import "./lib/setup_sharify"
+
+import compression from "compression"
+import express from "express"
+import favicon from "serve-favicon"
+import glob from "glob"
+import path from "path"
+import sharify from "sharify"
+import siteAssociation from "artsy-eigen-web-association"
+
+function staticAssetMiddlewares(app) {
```

Comment:
Since `express` is a singleton is it necessary to pass it as an argument?

Reply:
What do you mean by singleton in this case? Executing `express()` creates you a new object. Regardless, a second refactoring would clean this up.